### PR TITLE
additional command needed for compililing packages in maintainer(not …

### PR DESCRIPTION
**Pull request by nuschpl**
…developer) mode

### CODE REVIEW

**Comment by iceman1001** [comment]
@doegox  and same with this one.  It just an textual instruction.

File: README [hunk 1]
```diff
@@ -3,6 +3,7 @@ README
 
 Compiling:
     automake
+    autoreconf -i
```
Comment (smortex) [inline README]: `autoreconf` will launch `automake` and `autoconf` so they are redundant.  Better only keep `autoreconf`, followed by `./configure` and `make` :wink:

**Comment by nuschpl** [comment]
Any way will be better than current broken

śr., 31.01.2018, 12:18 użytkownik Romain Tartière <notifications@github.com>
napisał:

**Comment by iceman1001** [comment]
In what why is current broken?

**Comment by smortex** [comment]
#57 was merged

**Pull request closed without merging**